TestGHACtion/ConsoleApp1/Program.cs

```diff
@@ -1,2 +1,50 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+
+public class Calculator
+{
+    // Add method
+    public static double Add(double num1, double num2)
+    {
+        return num1 + num2;
+    }
+
+    // Subtract method
+    public static double Subtract(double num1, double num2)
+    {
+        return num1 - num2;
+    }
+
+    // Multiply method
+    public static double Multiply(double num1, double num2)
+    {
+        return num1 * num2;
+    }
+
+    // Divide method
+    public static double Divide(double num1, double num2)
+    {
+        if (num2 == 0)
+        {
+            throw new ArgumentException("Cannot divide by zero.");
+        }
+
+        return num1 / num2;
+    }
+
+    // Main method for testing
+    public static void Main()
+    {
+        // Example usage
+        double resultAdd = Add(5, 3);
+        Console.WriteLine("Addition: " + resultAdd);
+
+        double resultSubtract = Subtract(8, 3);
+        Console.WriteLine("Subtraction: " + resultSubtract);
+
+        double resultMultiply = Multiply(4, 6);
+        Console.WriteLine("Multiplication: " + resultMultiply);
+
+        double resultDivide = Divide(9, 3);
+        Console.WriteLine("Division: " + resultDivide);
```

Comment:
The provided code can be refactored in several ways to improve readability and maintainability. Here are some recommendations:

- Move the arithmetic methods into a separate class or interface like `IMathematicalOperations`, which is preferable in the long run.
- Separate concerns by moving the main logic from the `Program.cs` file into its own entry point class or program class.
- Consider using a logging framework instead of directly printing to the console in the `Main` method.
- Remove magic numbers by replacing them with named constants.
- Use a dependency injection container to manage dependencies between objects.

The refactored `Program.cs` might look something like this:

```csharp
using System;
using Microsoft.Extensions.Configuration;
using Microsoft.Extensions.Logging;
using Microsoft.Extensions.Hosting;

namespace TestGHACtion.ConsoleApp1 {
    public class Program {
        private readonly ILogger _logger;
        private readonly IMathOperations _mathOps;
        
        public Program(ILogger<Program> logger, IMathOperations mathOps):
        {
            _logger = logger;
            _mathOps = mathOps;
        }
        
        public static void Main(string[] args)
        {
            var host = Host.CreateDefaultBuilder()
                .ConfigureLogging()
                .Use<|im_start|>Serialization.
                .Build()
            
            using (var scope = host.Services.CreateScope()) {
                var services = scope.ServiceProvider;
                _logger = services.GetRequiredService<ILogger<Program>>();
                _mathOps = services.GetRequiredService<IMathOperations>();
                ;
                TryRun(args).Wait();
            }
        }
         
        private async Task TryRun(string[] args) {
            try {
                // Example usage
                double resultAdd = await _mathOps.Add(5, 3);
                _logger.LogInformation("Addition: " + resultAdd);

                double resultSubtract = await _mathOps.Subtract(8, 3);